docs/src/pages/components/alert/alert-de.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: Hinweis React-Komponente
 components: Alert, AlertTitle
+githubLabel:
+  component: Hinweis
```

Comment:
Two issues 1. the format is wrong 2. it shouldn't be translated

Reply:
For 1. I did it wrong when introducing this key. It's note valid markdown. I should have added quotes. For 2. I guess once we fix 1. We can make the keys as hidden.

Reply:
I have open #22629 to help on the matter